adafruit_minimqtt/adafruit_minimqtt.py

```diff
@@ -1,28 +1,11 @@
-# The MIT License (MIT)
-#
-# Copyright (c) 2019 Brent Rubell for Adafruit Industries
+# SPDX-FileCopyrightText: 2019 Brent Rubell for Adafruit Industries
 #
+# SPDX-License-Identifier: MIT
+
 # Original Work Copyright (c) 2016 Paul Sokolovsky, uMQTT
```

Comment:
@dherrada do these other copyrights need to be added to the `SPDX-FileCopyrightText`?

Reply:
@brentru I'm not actually sure. I think they might.